bundles/org.openhab.binding.teslapowerwall/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="teslapowerwall"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="teslapowerwall">
```

Comment:
```suggestion
	<thing-type id="tesla-powerwall">
```

Can you change the thing and channel ids and typeids to align with the naming convention? 
For reference: https://www.openhab.org/docs/developer/guidelines.html#naming-convention

Also check all Labels, they should be short (< 23 chars, 2-3 words) and capitalized.

Reply:
Hopefully the latest is better.